dnf/plugin.py

```diff
@@ -133,6 +134,21 @@ def _run_init(self, base, cli=None):
     def _unload(self):
         del sys.modules[DYNAMIC_PACKAGE]
 
+    def unload_removed_plugins(self, transaction):
+        erased = set([package.name for package in transaction.remove_set])
+        if not erased:
+            return
+        installed = set([package.name for package in transaction.install_set])
+        transaction_diff = erased - installed
+        if not transaction_diff:
+            return
+        files_erased = set()
```

Comment:
probably will be faster on big filesets/packagesets
`set(itertools.chain(pkg.files for pkg in transaction.remove_set) if pkg.name in transaction_diff))`